From f09de07d0c214b8170b81a0728905b0d98998268 Mon Sep 17 00:00:00 2001
From: Denys Drozdov <denys.drozdov@toradex.com>
Date: Wed, 12 May 2021 12:30:52 +0300
Subject: [PATCH] arm64: dts: apalis-imx8: drop m4 and rpmsg from
 reserved-memory

m4 and rpmsg submodes should not be reserved
for use in Easy Installer FIT image

Related-to: TEI-776
Signed-off-by: Denys Drozdov <denys.drozdov@toradex.com>
---
 arch/arm64/boot/dts/freescale/imx8-apalis-v1.1.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8-apalis-v1.1.dtsi b/arch/arm64/boot/dts/freescale/imx8-apalis-v1.1.dtsi
index a2ef38067313..3c6a5ca15743 100644
--- a/arch/arm64/boot/dts/freescale/imx8-apalis-v1.1.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8-apalis-v1.1.dtsi
@@ -182,14 +182,6 @@
 		 * Shouldn't be used at A core and Linux side.
 		 *
 		 */
-		m4_reserved: m4@0x88000000 {
-			no-map;
-			reg = <0 0x88000000 0 0x8000000>;
-		};
-		rpmsg_reserved: rpmsg@0x90000000 {
-			no-map;
-			reg = <0 0x90200000 0 0x200000>;
-		};
 		decoder_rpc: decoder_rpc@0x92000000 {
 			no-map;
 			reg = <0 0x92000000 0 0x200000>;
-- 
2.17.1

